### ISSUE TITLE
"UnicodeDecodeError" when pip install

### ISSUE THREAD

**Issue by curiositer** [flask-dashboard/Flask-MonitoringDashboard#231]
**Describe the bug**
when I try to use pip install Flask-MonitoringDashboard,it happened.
File "C:\...\flask-monitoringdashboard\setup.py", line 11, in get_description
        info = readme.read()
    UnicodeDecodeError: 'gbk' codec can't decode byte 0xa2 in position 1571: illegal multibyte sequence
   ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\...\pip-install-c4devic4\Flask-MonitoringDashboard\

**To Reproduce**
1.pip install Flask-MonitoringDashboard
2. See error

or 
1.Download the source
2. Add ".encode("utf-8")" after "readme.read()"
3.See error

**Expected behavior**
I want to install this.

**Screenshots**
![123456](https://user-images.githubusercontent.com/34828878/56089951-4165ad80-5ecd-11e9-8d28-a234eda8edf6.png)

**Desktop (please complete the following information):**
 - OS: [windows 10 professional]

**Additional context**
so I know it is an UnicodeDecodeError,but I cannot deal with it,thanks so much!

**Comment by bogdanp05** [comment]
Hi @curiositer,

I wasn't able to replicate this. I used the following settings on my system:
- Windows 10 64 bit
- virtual environment as described [here](https://flask-monitoringdashboard.readthedocs.io/en/latest/installation.html#configuring-the-virtual-environment-optional)
- Python 3.7.3

Running `pip install Flask-MonitoringDashboard` installed the package successfully. Could you give us more details about your system, so we can try to replicate the error?

Regards,
Bogdan

**Comment by curiositer** [comment]
Oh,thanks! I have solved the problem.

I used pip install flask-Monitoring-Dashboard, and it succeed.
Without of using "-",when I used pip install Flask-MonitoringDashboard,the error happened.

Thanks a lot, but I don'n know why,maybe it doesn't match with the "gbk" unicode?

I used the following settings on my system:
- Windows 10 64 bit
- Python 3.6.8 :: Anaconda, Inc.

Haha, my English is not so well,thank you for your reply.

Regards,
John

**Comment by bogdanp05** [comment]
> I used pip install flask-Monitoring-Dashboard, and it succeed.

I'm afraid you installed something else. You installed [this](https://pypi.org/project/flask-monitoring-dashboard/), which is a very old and no longer supported version of the Flask Monitoring Dashboard package, whereas the new version is [this one](https://pypi.org/project/Flask-MonitoringDashboard/2.1.4/).

Could you try to run `pip install Flask-MonitoringDashboard` inside a virtual environment? Also, what does `sys.getdefaultencoding()` return on your system?

> Thanks a lot, but I don'n know why,maybe it doesn't match with the "gbk" unicode?

It's probably trying to use GBK to read a file encoded with UTF-8

**Comment by curiositer** [comment]
Oh,thanks for warning me,I must install the old version.

When I run this:
`
import sys
sys.getdefaultencoding()
`
The result is: "'utf-8'"

And I run "pip install Flask-MonitoringDashboard",I got the same error as I had met.(The screenshot I had posted upstairs.)
Then I tried the url you have given to me,download it, unzip, and `python setup.py install`,but got the same error.
So how to deal with it?Thanks!

**Comment by bogdanp05** [comment]
In the `setup.py` file, try to replace line 10:
`with open(loc + '/README.md') as readme:`
with this:
`with open(loc + '/README.md', encoding='utf-8') as readme:`
Let me know if this fixes the error.

**Comment by curiositer** [comment]
Thanks so much,I installed it successfully,I had tried change the read() encode,but it doesn't exist.
I forget the open function can change the encoding.
It seems to have another error,I think maybe because of the Internet,and then I tried to use `pip install Flask-MonitoringDashboard`, finally it succeed.
The error code below:

error: Download error for https://files.pythonhosted.org/packages/ed/29/d97b6252591da5f8add0d25eecda296ea72729a0aad7998edba1981b47c8/numpy-1.16.2-cp36-cp36m-win_amd64.whl#sha256=d20c0360940f30003a23c0adae2fe50a0a04f3e48dc05c298493b51fd6280197: EOF occurred in violation of protocol (_ssl.c:852)

I have installed the package successfully.Thanks a lot!
